draft-ietf-taps-interface.md

```diff
@@ -1378,19 +1504,19 @@ does provide the following guarantees about the ordering of operations:
 
 # Transport Properties {#transport-props}
 
-Having discussed Transport Properties in {{connection-props}} and listed a
-number of Message Properties ("Ordered", "Idempotent" etc.) in {{message-props}},
+Having discussed Selection Properties in {{connection-props}}, Connection Properties {{connection-props}} and,  Message Properties ({{message-props}}),
 we now provide a complete overview of a transport system's defined Transport Properties.
 
-Properties are structured in two ways:
-
- - By how they influence the transport system, which leads to a classification
-   into "Selection Properties", "Protocol Properties", "Control Properties" and "Intents".
- - By the object they can be applied to: Preconnections, see {{connection-props}},
-   Connections, see {{introspection}}, and Messages, see {{message-props}}.
+Transport Properties are structured by the context they can apply on: 
```

Comment:
Why change "the object they apply on" to "context they apply on"? I think "object" was clearer.